scripts/hudson/narayana.sh

```diff
@@ -521,10 +522,17 @@ function rts_tests {
 function lra_tests {
   echo "#0. LRA Test"
   ./build.sh install -pl ArjunaCore/arjunacore,rts -am -DskipTests -Pcommunity "$@"
-  # we can't use 'mvn -f' option beacuse of Thorntail plugin issue THORN-2049
+  # we can't use 'mvn -f' option beacuse of Thorntail plugin issue QUARKUS-2049
```

Comment:
I think this change will be wrong as the QUARKUS issues are under github and the plugin would not have the same issue under the same number.

Reply:
Thanks for spotting this Ondra. A huge oversight probably due to "find and replace".